tests/libstorage/datavolume.go

```diff
@@ -27,124 +27,21 @@ import (
-	name := "test-datavolume-" + rand.String(12)
-	quantity, err := resource.ParseQuantity(size)
-	util.PanicOnError(err)
-	dataVolume := &v1beta1.DataVolume{
-		ObjectMeta: v12.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: v1beta1.DataVolumeSpec{
-			Source: &dataVolumeSource,
-			PVC: &v1.PersistentVolumeClaimSpec{
-				AccessModes: []v1.PersistentVolumeAccessMode{accessMode},
-				VolumeMode:  &volumeMode,
-				Resources: v1.ResourceRequirements{
-					Requests: v1.ResourceList{
-						"storage": quantity,
-					},
-				},
-				StorageClassName: &storageClass,
-			},
-		},
-	}
-
-	dataVolume.TypeMeta = v12.TypeMeta{
-		APIVersion: "cdi.kubevirt.io/v1beta1",
-		Kind:       "DataVolume",
-	}
-
-	return dataVolume
-}
-
-func NewDataVolumeWithRegistryImportInStorageClass(imageUrl, namespace, storageClass string, accessMode v1.PersistentVolumeAccessMode, volumeMode v1.PersistentVolumeMode) *v1beta1.DataVolume {
-	var size string
-	switch imageUrl {
-	case cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskFedoraTestTooling), cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskFedoraRealtime):
-		size = cd.FedoraVolumeSize
-	default:
-		size = "512Mi"
-	}
-	// Uses node cache, does not require extra scratch space PVC
-	pullMethod := v1beta1.RegistryPullNode
-	dataVolumeSource := v1beta1.DataVolumeSource{
-		Registry: &v1beta1.DataVolumeSourceRegistry{
-			URL:        &imageUrl,
-			PullMethod: &pullMethod,
-		},
-	}
-	return newDataVolume(namespace, storageClass, size, accessMode, volumeMode, dataVolumeSource)
-}
-
-func NewBlankDataVolume(namespace, storageClass, size string, accessMode v1.PersistentVolumeAccessMode, volumeMode v1.PersistentVolumeMode) *v1beta1.DataVolume {
-	dataVolumeSource := v1beta1.DataVolumeSource{
-		Blank: &v1beta1.DataVolumeBlankImage{},
-	}
-	return newDataVolume(namespace, storageClass, size, accessMode, volumeMode, dataVolumeSource)
-}
-
-func NewDataVolumeWithPVCSource(sourceNamespace, sourceName, targetNamespace string, accessMode v1.PersistentVolumeAccessMode) *v1beta1.DataVolume {
-	sc, exists := GetRWOFileSystemStorageClass()
-	if accessMode == v1.ReadWriteMany {
-		sc, exists = GetRWXFileSystemStorageClass()
-	}
-	if !exists {
-		ginkgo.Skip("Skip test when Filesystem storage is not present")
-	}
-	return newDataVolumeWithPVCSourceWithStorageClass(sourceNamespace, sourceName, targetNamespace, sc, "1Gi", accessMode)
-}
-
-func newDataVolumeWithPVCSourceWithStorageClass(sourceNamespace, sourceName, targetNamespace, storageClass, size string, accessMode v1.PersistentVolumeAccessMode) *v1beta1.DataVolume {
-	dataVolumeSource := v1beta1.DataVolumeSource{
-		PVC: &v1beta1.DataVolumeSourcePVC{
-			Namespace: sourceNamespace,
-			Name:      sourceName,
-		},
-	}
-	volumeMode := v1.PersistentVolumeFilesystem
-	return newDataVolume(targetNamespace, storageClass, size, accessMode, volumeMode, dataVolumeSource)
-}
+const CirrosPVCSize = "512Mi"
```

Comment:
What about putting this here and naming it similar to `cd.FedoraVolumeSize`? And also adding a `cd.BlankVolumeSize` or `cd.EmptyVolumeSize` for `64Mi`?

https://github.com/kubevirt/kubevirt/blob/8a9f939b7619bef3499989e6635434d55005842c/tests/containerdisk/containerdisk.go#L42

Reply:
:+1000